python/lsst/sims/maf/batches/scienceRadarBatch.py

```diff
@@ -232,21 +234,43 @@ def scienceRadarBatch(colmap=None, runName='opsim', extraSql=None, extraMetadata
 
     displayDict['subgroup'] = 'N stars'
     slicer = slicers.HealpixSlicer(nside=nside, useCache=False)
-    sum_stats = [metrics.SumMetric(metricName='Total N Stars')]
+    sum_stats = [metrics.SumMetric(metricName='Total N Stars, crowding')]
+    for f in filterlist:
+        stellar_map = maps.StellarDensityMap(filtername=f)
+        displayDict['order'] = filterorders[f]
+        displayDict['caption'] = 'Number of stars in %s band with an measurement error due to crowding ' \
+                                 'of less than 0.2 mag' % f
+        # Configure the NstarsMetric - note 'filtername' refers to the filter in which to evaluate crowding
+        metric = metrics.NstarsMetric(crowding_error=0.2, filtername=f,
```

Comment:
Can you add ignore_crowding=False to the arguments here?  I am guessing that's the default now, but it might not always be .. and by making it explicit it's easier to see the difference here vs. the metric below (which also has crowding_error=0.2, which make it harder to spot the difference).